frontend/src/components/ProjectUserRow.vue

```diff
@@ -44,20 +49,20 @@ limitations under the License.
               <v-icon>mdi-pencil</v-icon>
             </v-btn>
           </template>
-          <span>Update User</span>
+          <span>Change user roles</span>
         </v-tooltip>
       </v-list-item-action>
       <v-list-item-action v-if="canManageMembers" class="ml-1">
         <v-tooltip top>
           <template v-slot:activator="{ on }">
             <div v-on="on">
-              <v-btn :disabled="isTechnicalContact" icon color="red" @click.native.stop="onDelete">
-                <v-icon>mdi-delete</v-icon>
+              <v-btn :disabled="isowner" icon color="red" @click.native.stop="onDelete">
+                <v-icon>mdi-close</v-icon>
               </v-btn>
             </div>
           </template>
-          <span v-if="isTechnicalContact">This user is set as technical contact</span>
-          <span v-else>Delete User</span>
+          <span v-if="isowner">This user is set as owner</span>
```

Comment:
when using the UI I was rather expecting something like this instead: `You can't remove project owners from the project. You can change the project owner on the administration page.`

Reply:
Yes, I think I had it that way but somehow lost the change. I will change it to your proposal.